### PR TITLE
Buffs the Bioterror Chem Sprayer

### PR DESCRIPTION
**Pull request by Iamgoofball**
The Bioterror Chem Sprayer is a fun idea on paper, but the reagents in it are kinda shit. It's one of the least bought items because it's pretty useless.

It now has the following reagents:
Sodium Thiopental, for stamina and knockout
Coniine, for Toxin and Oxygen
Venom, for scaling Toxin, Brute, and Histamine Decay
Condensed Capsaicin, for an instant stun on application
Initropidril, for the heart attacks
Polonium, for the severe radiation

Basically:
"OH GOD MY EYES THE PEPPERSPRAY OH FUCK I CAN'T BREATHE AUGH MY HEART STOPPED I JUST GREW A NEW LEG OH GOD MY SKIN'S PEELING OFF AND I FEEL TIRED AS FUCK"

All in all, this should make the Bioterror sprayer actually worth the points investment, and make it more fun to use as an operative.

### CODE REVIEW

**Comment by Cheridan** [comment]
cool

**Comment by SconesC** [comment]
Finally

**Comment by Lumipharon** [comment]
Sprayer used to be better, but someone made a change, that among other things, made it weaker over range.
So back when Aranesp was the ultimate death chem, I could use the sprayer to kill people at close range almost instantly, but if I used it on someone on the otherside of the screen, it would barely even effect them.

As far as I know it's the only weapon that not only has a max range (screen length) but also gets weaker over distance.

**Comment by Supermichael777** [comment]
point away from face
